### PR TITLE
Use plain files instead of pipes to capture child process output

### PR DESCRIPTION
**Pull request by AlexDenisov**

### CODE REVIEW

File: lib/ForkProcessSandbox.cpp [hunk 1]
```diff
@@ -35,23 +35,38 @@ pid_t mullFork(const char *processName) {
...
   }
-
-  if (pipe(stderr_file_descriptor) == -1) {
-    perror("stderr_pipe");
-    exit(1);
+  if (unlink(filename) == -1) {
+    perror("unlink");
   }
 
+  fseek(file, 0, SEEK_END);
```
Comment (stanislaw) [inline lib/ForkProcessSandbox.cpp]: Let me get this right: you first delete the file and then read from it? Is this a normal practice?

Comment (stanislaw) [reply lib/ForkProcessSandbox.cpp]: Your method should be smth like: `unlinkFileAndRead` 😄

Comment (AlexDenisov) [reply lib/ForkProcessSandbox.cpp]: The man page would explain it better than I can
https://linux.die.net/man/2/unlink

Comment (stanislaw) [reply lib/ForkProcessSandbox.cpp]: Good.

**Pull request merged**